purple-pill/templates/menu.html

```diff
@@ -28,14 +31,14 @@
         </div>
         <div class="lm-body" ng-if="data.localmenu.help">
             <div class="lm-button"><a href="https://github.com/metapolator/metapolator" target="_blank">Github Project</a></div>
-            <div class="lm-button"><a title="Browse existing feature requests and bug reports on our Github issue tracker" href="https://github.com/metapolator/metapolator/issues/new" target="_blank">Github Issues</a></div>
+            <div class="lm-button"><a title="Browse existing feature requests and bug reports on our Github issue tracker" href="https://github.com/metapolator/metapolator/issues" target="_blank">Github Issues</a></div>
             <div class="lm-button"><a title="To report a bug or request a feature, create a new issue in our Github issue tracker" href="https://github.com/metapolator/metapolator/issues/new" target="_blank">New Issue</a></div>
             <div class="lm-divider"></div>
-            <div class="lm-button"><a title="To ask questions about usage or anything in general, post on our G+ Community page" href="https://metapolator.com" target="_blank">metapolator.com</a></div>
-            <div class="lm-button"><a title="To ask questions about usage or anything in general, post on our G+ Community page" href="https://twitter.com/metapolator" target="_blank">@metapolator</a></div>
-            <div class="lm-button"><a title="To ask questions about usage or anything in general, post on our G+ Community page" href="https://plus.google.com/communities/110027004108709154749" target="_blank">+metapolator</a></b></div>
+            <div class="lm-button"><a title="To ask questions about usage or anything in general, post on our G+ Community page" href="https://metapolator.com" target="_blank">Metapolator.com</a></div>
+            <div class="lm-button"><a title="To ask questions about usage or anything in general, post on our G+ Community page" href="https://twitter.com/metapolator" target="_blank">@Metapolator</a></div>
```

Comment:
I think the URL and twitter should be all lower case, but plus is okay capitalised